src/debugger/Debugger.cpp

```diff
@@ -505,6 +505,32 @@ bool Debugger::processIncomingMessages(ExecutionState* state, ByteCodeBlock* byt
 
     while (receive(buffer, length)) {
         switch (buffer[0]) {
+        case ESCARGOT_DEBUGGER_CLIENT_SOURCE: {
+            uint32_t size;
+            char* data;
+            memcpy(&size, buffer + 1, sizeof(uint32_t));
+            data = (char*)GC_MALLOC_ATOMIC(size);
+            char* ptr = data;
+            char* end = data + size;
+            length = ESCARGOT_DEBUGGER_MAX_MESSAGE_LENGTH - 1 - sizeof(uint32_t);
+            memcpy(ptr, buffer + 1 + sizeof(uint32_t), length);
+            ptr += length;
+            do {
+                if (receive(buffer, length)) {
+                    memcpy(ptr, buffer + 1, length - 1);
+                    ptr += length - 1;
+                } else if (!enabled()) {
+                    return false;
+                }
+            } while (ptr < end);
+            uint32_t sourceNameLen = strlen(data);
+            m_clientSourceName = new Latin1String(data, sourceNameLen);
+            m_clientSourceData = new Latin1String(data + sourceNameLen + 1, size - sourceNameLen - 1);
```

Comment:
This part handles the type of source name and source code as one byte string. But what if the type of source code is two byte string?